overlord/snapstate/flags.go

```diff
@@ -75,8 +75,8 @@ type Flags struct {
 	// IsAutoRefresh is true if the snap is currently auto-refreshed
 	IsAutoRefresh bool `json:"is-auto-refresh,omitempty"`
 
-	// IsContinuedAutoRefresh is true if this is a continued auto-refresh
-	IsContinuedAutoRefresh bool `json:"is-continued-auto-refresh,omitempty"`
+	// IsContinuedRefresh is true if this is a continued refresh
+	IsContinuedRefresh bool `json:"is-continued-auto-refresh,omitempty"`
```

Comment:
not sure we need this change of terminology the starting point is still an auto-refresh no?

Reply:
Yes, this was part of a change I later reverted. I miss this bit but I'll revert it now